Examples/Modules/Chapter 12/Ex12_17/Ex12_17.cpp

```diff
@@ -0,0 +1,59 @@
+// Using a linked list
```

Comment:
I couldn't get this Ex12_17 to compile.
After some workaround for VC++, I'm left with:
```
1>F:\PVW\beginning-cpp20\Examples\Modules\Chapter 12\Ex12_17\Truckload.cpp(9,32): error C2061: syntax error: identifier 'Package'
1>F:\PVW\beginning-cpp20\Examples\Modules\Chapter 12\Ex12_17\Truckload.cpp(9,32): error C2143: syntax error: missing ';' before '{'
1>F:\PVW\beginning-cpp20\Examples\Modules\Chapter 12\Ex12_17\Truckload.cpp(9,38): error C2143: syntax error: missing ';' before '}'
```

Reply:
Did you get it to compile?

Reply:
Don't remember. It indeed does not compile with the current preview of Visual Studio, no...